CHANGELOG.md

```diff
@@ -3,6 +3,28 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unpublished Version / DEV]
```

Comment:
The headline would need to be changed to `3.2`?

Reply:
Already resolved in https://github.com/nf-core/rnaseq/pull/673